### ISSUE TITLE
Have a list of user which can be raid

### ISSUE THREAD

**Issue by Foine** [ylorant/Hedgebot-admin#13]
It would be nice if a list of user that can be raid.

This list can be extract from the auto-host list ?

**Comment by RomainOdeval** [comment]
Technically, you can raid any Twitch channel you want, like "/raid" genuine Twitch command.
So i strongly not recommand to have this kind of "frozen" list.

But it will be interesting to have an auto-complete suggestion from auto-host list and have possibility to write a channel name not included from this list.

**Comment by ylorant** [comment]
Yeah, the original request Foine suggested to me vocally was to have more of a suggestion list than a set list that you cannot customize upon.
